src/Sylius/Behat/Context/Setup/ShippingContext.php

```diff
@@ -109,11 +109,19 @@ public function theStoreShipsEverywhereForFree()
     public function theStoreShipsEverywhereForFreeForAllChannels(array $channels)
     {
         foreach ($this->zoneRepository->findAll() as $zone) {
-            $shipment = $this->createShippingMethod('Free', null, null, $zone, 'en', ['amount' => 0], DefaultCalculators::FLAT_RATE, true, false);
-
-            foreach ($channels as $channel) {
-                $shipment->addChannel($channel);
-            }
+            $configuration = $this->getConfigurationByChannels($channels);
+            $this->createShippingMethod(
```

Comment:
As @pamil suggest, we have to start using example factories here. But it can be adjusted later